public/data/cardOptionsWithRaid.json

```
@@ -477,6 +482,24 @@
       ],
       "has_special_style": true,
       "rarities": []
+    },
+    {
+      "id": 14,
+      "short_name": "Dragon",
+      "name": "Dragon",
+      "supertypes": [
+        1,
+        4
+      ],
+      "has_subtypes": true,
+      "subtype_optional": false,
+      "is_energy": true,
+      "rarities": [
+        1,
+        2,
+        3,
+        4
+      ]
```

Let's change this in order to be consistent with other energy types
```suggestion
      "rarities": []
```